Reinstate Services

LiviaFResende opened the pull request:
## Rationale

This is for reinstate the Services Page on the Website

## Advice for Reviewers & Testing Notes

- Review code for file "src\dark\PortfolioLanding.jsx"
- The file above is linked to the "ServiceList" file that was ready to be reinstated.

## Screenshots:

![image](https://user-images.githubusercontent.com/91722122/200460696-93de22b1-747b-42af-a35a-fedcd3c4cbe8.png)

## Task Name

 - #48 

## Linting Checklist
- [ ] No commented code
- [ ] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

LiviaFResende committed:
reinstate Services

davidtaing commented on src/dark/PortfolioLanding.jsx:
```diff
@@ -64,6 +64,14 @@ const PortfolioLanding = () => {
                 </div>
             </div>
             {/* End Slider Area   */} 
+            
+                {/* Start Service Area  */}
+                <div className="service-area ptb--120 bg_color--5">
+                   <div className="container">
+                        <ServiceList />
+                   </div>
+                </div>
+                {/* End Service Area  */}
```
Hi @LiviaFResende,
It looks like the Services section is broken. You're not in trouble, but please make sure to check your changes on your local machine before sending in a PR.

![image](https://user-images.githubusercontent.com/8443215/200461412-8336937f-89fd-466f-a9fe-240fb93cbb1b.png)

Can you please copy and paste the section code from this file? The red lines are the lines that were removed.

https://github.com/codesydney/lloydconsulting/commit/09e054d405de592c7f458b162329b4c156f8ca85#diff-be3c8e4a245b8ba5180f3063a19dfce031d1c49760e7008d64660de11da90ac6

davidtaing replied:
I'll fix this one for you. You don't have to worry about this anymore.

davidtaing committed:
reinstate: services section

davidtaing committed:
fix: service card width

Service card width was set to auto for some reason. This change adds
a 100% width.

davidtaing committed:
chore: remove unused service item

davidtaing committed:
Merge branch 'main' into NewServices

davidtaing committed:
reinstate Services navlink
